webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -28,4 +53,328 @@ def calculate_pauschbetrag(has_pflegegrad=False, disability_degree=None, has_mer
     elif disability_degree >= 30:
         return 620
     elif disability_degree >= 20:
-        return 384
\ No newline at end of file
+        return 384
+
+    return 0
+
+
+class DisabilityModel(BaseModel):
+    person_a_has_disability: Optional[str]
+    person_a_has_pflegegrad: Optional[str]
+    person_a_disability_degree: Optional[int]
+    person_a_has_merkzeichen_bl: Optional[bool]
+    person_a_has_merkzeichen_tbl: Optional[bool]
+    person_a_has_merkzeichen_g: Optional[bool]
+    person_a_has_merkzeichen_ag: Optional[bool]
+    person_a_has_merkzeichen_h: Optional[bool]
+
+    person_b_has_disability: Optional[str]
+    person_b_has_pflegegrad: Optional[str]
+    person_b_disability_degree: Optional[int]
+    person_b_has_merkzeichen_bl: Optional[bool]
+    person_b_has_merkzeichen_tbl: Optional[bool]
+    person_b_has_merkzeichen_g: Optional[bool]
+    person_b_has_merkzeichen_ag: Optional[bool]
+    person_b_has_merkzeichen_h: Optional[bool]
+
+    @root_validator()
+    def convert_disability_degree_to_int(cls, values):
+        values['person_a_disability_degree'] = int(values['person_a_disability_degree']) \
+            if values['person_a_disability_degree'] \
+            else None
+        values['person_b_disability_degree'] = int(values['person_b_disability_degree']) \
+            if values['person_b_disability_degree'] \
+            else None
+        return values
+
+
+class PersonAHasPflegegradSetPrecondition(DisabilityModel):
+    _step_to_redirect_to = StepFamilienstand.name  # TODO: StepPersonAMerkzeichen.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_not_filled_pflegegrad')
+
+    person_a_has_pflegegrad: str
```

Comment:
Why do we have to add this Precondition? Couldn't this be the hasMerkzeichenData Precondition?

Reply:
Or should we use this for the MerkzeichenPrecondition because the `has_pflegegrad` is always required